chandra_aca/planets.py

```diff
@@ -586,3 +591,132 @@ def get_planet_chandra_horizons(
     del dat["null3"]
 
     return dat
+
+
+@dataclass
+class EarthBoresightAngles:
+    """Earth boresight angles from Chandra
+
+    Attributes
+    ----------
+    times : CxoTime
+        Times
+    earth_limb_angle : ndarray
+        Earth limb angle (deg)
+    earth_open_angle : ndarray
+        Earth opening angle (deg)
+    earth_center_angle : ndarray
+        Earth center angle (deg)
+    """
+
+    times: CxoTime
+    earth_limb_angle: np.ndarray
+    earth_open_angle: np.ndarray
+    earth_center_angle: np.ndarray
+
+
+def get_earth_boresight_angle(start: CxoTimeLike, stop: CxoTimeLike):
+    """Calculate Earth boresight angle from Chandra.
+
+    The key calculations were adapted from acis_taco.calc_earth_vis() so there is some
+    heritage.
```

Comment:
Sorry, I didn't have anything against the parameters. What I meant is that, *from this moment on* (as opposed to _from this line on_) the body of the docstring adds no new information, because this phrase is intended for a reviewer, and is pretty useless for future users. I'm referring to the phrase "The key calculations were adapted from acis_taco.calc_earth_vis() so there is some heritage."

Reply:
OK. I'll make that a code comment. It will definitely be helpful to me in 2 years when I wonder where these equations came from.

Reply:
Done.